build-info-extractor-gradle/src/test/java/org/jfrog/gradle/plugin/artifactory/GradlePluginTest.java

```diff
@@ -49,11 +51,14 @@ protected void cleanup() throws IOException {
 
     @DataProvider
     private Object[][] gradleVersions() {
-        return new String[][]{{"4.10.3"}, {"5.6.4"}, {"6.3"}};
+        return new String[][]{{"4.10.3"}, {"5.6.4"}, {"6.9"}, {"7.5.1"}};
     }
 
     @Test(dataProvider = "gradleVersions")
     public void configurationsTest(String gradleVersion) throws IOException {
+        if (MIN_VERSION_WITHOUT_CONFIGURATIONS.isAtLeast(new Version("7"))) {
```

Comment:
Wouldn't this condition always be truue? `MIN_VERSION_WITHOUT_CONFIGURATIONS` is initializxed to `new Version("7"))`.
The same applies to the `deprecatedCiServerTest` method.